base/applications/network/nslookup/nslookup.c

```diff
@@ -792,26 +792,30 @@ int main( int argc, char* argv[] )
     /* We don't know how long of a buffer it will want to return. So we'll
        pass an empty one now and let it fail only once, instead of guessing. */
     Status = GetNetworkParams( pNetInfo, &NetBufLen );
-    if( Status == ERROR_BUFFER_OVERFLOW )
+
+    if( Status != ERROR_BUFFER_OVERFLOW ) {
```

Comment:
The rest of this file appears to put braces on their own lines. Would be good to stay consistent.

Reply:
True. Fixed in the updated version.